Source/SwiftCloudant/HTTP/HTTPInfo.swift

```diff
@@ -0,0 +1,34 @@
+//
+//	HTTPInfo.swift
```

Comment:
indention on the filename line is like 2x vs 1x or something 😅

Reply:
@jpswann288-- nvm, this was my bad, I used a tab on the filename line vs two spaces. Don't worry about it, I am pushing a fix on #8